ios/sdk/src/LaunchOptions.m

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright @ 2018-present Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Intents/Intents.h>
+
+#import <React/RCTBridge.h>
+#import <React/RCTBridgeModule.h>
+
+@interface LaunchOptions : NSObject<RCTBridgeModule>
+
+@property (nonatomic, weak) RCTBridge *bridge;
```

Comment:
This is just a clarifying question as I don't know how things work in Objective C. What is happening here ? Is it some React magic which will add bridge as a field to the LaunchOptions ? Does this magic happen when RTC_EXPORT_MODULE() is present ?

Reply:
This is also some React Native module stuff. By adding the property the module gets access to the bridge in order to be able to call stuff on it.